.spelling

```diff
@@ -37,6 +37,7 @@
 32-character
 32-port
 39.9kVA
+3m
```

Comment:
This seems like an easy enough typo to make in the docs that it seems preferable to just expand this to "Three meter" where it appears in the glossary. As much as possible I want to avoid adding items to the spelling file when they are things that could also easily be mistakes that we'd want to flag.

Failing that, I'd add this as a file-specific exception for the glossary file. But in this case the simplest path seems to just expand the abbreviation.

```suggestion
```